go-controller/pkg/ovn/egressip.go

```diff
@@ -1941,6 +1886,72 @@ func createDefaultNoRerouteServicePolicies(nbClient libovsdbclient.Client, v4Clu
 	return nil
 }
 
+// createDefaultRerouteToExternal is called only when IC=true. This function adds a "catch-all" kind of LRSR to ovn-cluster-router
+// 100.64.0.2               168.254.0.2 dst-ip
+// 100.64.0.3               168.254.0.3 dst-ip
+// 100.64.0.4               100.64.0.4 dst-ip
+// 10.244.0.0/24            168.254.0.2 dst-ip
+// 10.244.1.0/24            168.254.0.3 dst-ip
+// 10.244.2.0/24            100.64.0.4 src-ip
+// 10.244.0.0/16            100.64.0.4 src-ip ----> This is the reroute added to send all traffic that did not match earlier LRSR's to outside the cluster
+// This logic works under the assumption that we have all other paths covered via routes that exist with higher precedence prefix match
+// On first look it may seem like we are sending out traffic that doesn't "fit/match" other routes which is true, but the intent is that
+// if we don't know where to send the traffic within the cluster, we make it leave the cluster (we have a flow on br-ex that protexcts us and
+// drops it if its not supposed to be going outside). This is needed when IC=true to ensure traffc from the other node arriving at this remote node
+// does not get dropped. This removes the need for per-pod LRSR for EIP and ESVC adds a per-pod LRP on each egressNode to override this LRSR and send
+// it to it's management port. NOTE: Handle changes around this logic with care. This is being added intentionally.
+func createDefaultRerouteToExternal(nbClient libovsdbclient.Client, v4ClusterSubnet, v6ClusterSubnet []*net.IPNet, nodeName string) error {
```

Comment:
nit: calling this reroute might be a bit misleading, as this is just a route and reroute might be understood as a policy action

Reply:
changing it to `createDefaultRouteToExternal`